### PR TITLE
key mod

### PR DESCRIPTION
**Pull request by Artemis-chan**
fixes part 1 of #4, backported from osu mania as suggested by @Thesola10

### CODE REVIEW

**Comment by Thesola10** [comment]
wonder if there's a way to use custom graphics as mod icons

File: osu.Game.Rulesets.Diva/Mods/DivaModKey3.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Diva.Mods
+{
+	public class DivaModKey3 : DivaKeyMod
+	{
+		public override int KeyCount => 3;
+		public override string Name => "3 Button";
```
Comment (Thesola10) [inline osu.Game.Rulesets.Diva/Mods/DivaModKey3.cs:9]: should be "Three Button" for consistency

Actually make Button plural on all ModKey as well?

**Review by Thesola10 — CHANGES_REQUESTED** [review]

**Comment by Thesola10** [comment]
No functional problems upon testing tho

**Review by Thesola10 — APPROVED** [review]
naisu

**Comment by Artemis-chan** [comment]
> 
> 
> wonder if there's a way to use custom graphics as mod icons

It seems like they are using customs fonts  for the custom mod graphics ([source](https://github.com/ppy/osu/blob/ab49e84d170cf7376ca25e6f239306ccf840475a/osu.Game/Graphics/OsuIcon.cs#L8)).

**Comment by Thesola10** [comment]
Doesn't that mean we can introduce our own font (it's some compiled PNG table) and use that for mod icons?

**Comment by Artemis-chan** [comment]
i have looked into it, ([this is how the framework loads](https://github.com/ppy/osu-framework/blob/51c96583c08a6167a7048c6486604f6a5f5a966f/osu.Framework/Game.cs#L209))
I think if we use the background dependency loader to inject OsuGame(Base?) and add the font using `AddFont()`

**Pull request merged**